driver-core/src/test/resources/unified-test-format/transactions/mongos-pin-auto.json

```diff
@@ -2004,6 +2004,104 @@
         }
       ]
     },
+    {
```

Comment:
Let's revert this change (and the associated skip of the tests) to save Valentin the hassle of a merge conflict.